ui/src/app/components/cromwell-configuration-panel.spec.tsx

```diff
@@ -60,23 +66,6 @@ describe('CromwellConfigurationPanel', () => {
     disksApiStub = new DisksApiStub();
     registerApiClient(DisksApi, disksApiStub);
 
-    workspacesApiStub = new WorkspacesApiStub();
```

Comment:
I like how replacing the workspace api code with a prop in the cromwell configuration panel, you were able to get rid of this code in the test. I think that that makes the test more readable.

Reply:
Thanks, I completely agree